### PR TITLE
Updated README.

### PR DESCRIPTION
**Pull request by taphu**
See commits for comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,67 @@
...
+
+Sometimes, it is desirable to describe interaction with nodes that exist
+outside of your local environment, such as third-party apis or services. Some
+people like to specify these kinds of external services using fancy node
+styles.
+
+Example:
+
+    digraph {
+        "third-party-api" [style=dashed];
```
Comment (pyen) [inline README.md]: Is there any agreement/standardization on the fancy styles used? Browsing Eureka now yields instances of `[style=dashed]` but none other.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,67 @@
...
+## Participant specification.
+
+Each participating service instance must register a special Eureka metadata value
+containing the cartographer data for that service.
+
+The well-known Eureka metadata key is the value "cartography"
+
+The format of the cartographer data is an embedded graphviz DOT document
+describing the relationships between the participating service and other
+services with which it communicates.
```
Comment (pyen) [inline README.md]: I'd consider linking http://www.graphviz.org/content/dot-language or https://en.wikipedia.org/wiki/DOT_(graph_description_language) as a reference.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,67 @@
...
+people like to specify these kinds of external services using fancy node
+styles.
+
+Example:
+
+    digraph {
+        "third-party-api" [style=dashed];
+        "my-service" -> "third-party-api";
+        "my-service" -> "some-supporting-service";
+        "my-service" -> "eureka";
```
Comment (pyen) [inline README.md]: Is there a list of agreed-upon externalities? Have seen `eureka`, `redshift` and `dynamo`, but these seem to be agreed-upon as opposed to enforced labels.

File: README.md (hunk 1 above)
Comment (taphu) [inline README.md]: At SumAll, we have, by fait accompli, decided on `[style=dashed]` for external services. However, this is an open source project and I don't really want to make that kind of mandate here. I guess we need another document describing SumAll's use.

File: README.md (hunk 3 above)
Comment (taphu) [inline README.md]: Same as above.

File: README.md (hunk 2 above)
Comment (taphu) [inline README.md]: good idea. I will.

**Pull request merged**